src/components/layouts/layout.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+    <div class="layout">
+        <Navbar />
+        <slot />
+        <Footer />
+    </div>
+</template>
+  
+<script setup>
+import Navbar from '../Navbar.vue';
+import Footer from '../Footer.vue';
+</script>
+  
+<style scoped>
+.layout {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+}
+
+Footer {
```

Comment:
Geef het component een class en ga op basis daarvan stylen niet het component zelf stylen. En voor de rest van de styling hieronder.. Dat hoort in het component zelf niet in het layout component (Geld eigenlijk voor alle styling vanaf hier